### DISCUSSION TITLE
Is it "ok" to distribute a patch bundled with Camomile?

### DISCUSSION THREAD

**Discussion by unfamiliartunes** [pierreguillot/Camomile#263] [Q&A] [answered]
I'm almost done making a plugin with Camomile and Pd Vanilla. I've made ports for all operating systems, but I was wondering:

Can I distribute the plugin I made for free? Since I used Camomile for the VST3 side, the patch is bundled with versions of Camomile 1.0.7, and I'm unclear on the copyright law for that.

**Comment by pierreguillot** [comment, marked as answer]
Yes, you can distribute it for free. There are already the necessary copyrights in the About window of the plugin. I'm not a licensed expert but you can also add the copyrights and the license in a readme file shipped with the plugin if you want. In fact, you should just respect the GLP-3.0 license (that is inherited from Juce). Of course, if you can cite and link to the project in your announcement it would be really appreciated :)
